test/unit/subcommands/test_stop_subcommand.py

```diff
@@ -169,13 +169,15 @@ def terminate_only_master_process_successfully(pid, _):
         self._stop_subcommand.run(None)
 
         # Assert
+        expected_call_args_list = []
+        expected_call_args_list.append(call(slave_child_process.pid, signal.SIGTERM))
+        expected_call_args_list.append(call(slave_process.pid, signal.SIGTERM))
```

Comment:
Just curious why you chose to start with an empty list and append elements vs just doing a list literal? (I was guessing you chose that so it matched the semantics of the if-statement below, and thought if so then you could still make it match with a list literal by using `extend()` instead of `append()`.)